webapp/components/Upload/index.vue

```diff
@@ -41,6 +41,12 @@ export default {
       hover: false,
     }
   },
+  computed: {
+    avatarUrl() {
+      const { avatar } = this.user
+      return avatar && avatar.url
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-18T16:17:59Z" title="Tuesday, February 18th 2020, 5:17:59 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -41,6 +41,12 @@ export default {
       hover: false,
     }
   },
+  computed: {
+    avatarUrl() {
+      const { avatar } = this.user
+      return avatar && avatar.url
```


anybody else feel like using the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)? :sweat_smile:

it's still experimental technology but I'd like to see if there's a webpack plugin for it already...

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-19T08:45:41Z" title="Wednesday, February 19th 2020, 9:45:41 am +01:00">Feb 19, 2020</time>_

---


you want more fun configuring webpack? I got no problem with using it, although the `experimental` part makes me a bit nervous.